systemd/init

```diff
@@ -0,0 +1,51 @@
+#!/bin/busybox sh
+
+set -e
+
+# Init for Wolfi OS
+# - Environment variables:
+#     VERBOSE=1 for debugging
+#     CMDLINE=/path/to/file to override /proc/cmdline, useful for testing
+# - Mount /proc, /sys, /dev if not already mounted
+# - Allow for override of path to /init with init=/path/to/init on kernel /proc/cmdline
+# - Allow for uuid or device path for root filesystem with root= on kernel /proc/cmdline
+# - Mount root, if specified and found, switch_root to it, and run systemd as init
+# - Otherwise, try to run systemd as init in the initrd
+
+# Verbose debugging
+info() { [ "$VERBOSE" = "1" ] && printf "INFO: %s\n" "$*" 1>&2; }
+
+# Override /proc/cmdline path as environment variable, useful for testing
+[ -z "$CMDLINE" ] && CMDLINE="/proc/cmdline"
+info "cmdline = [$(cat $CMDLINE)]"
+
+# Setup our system mounts if needed
+mkdir -p /proc /sys /dev
```

Comment:
should we move the creation of the basic rootfs tree to the package itself? like in this commit: https://github.com/wolfi-dev/os/pull/24903/commits/9c383d2f35d5e1d9634d14003f051004cbd22609

Reply:
Hmm...maybe.  I like it here, especially with the "-p" option to make absolutely sure that these mount points exist, just before using them.  This feels very safe, and defensive, and also ensures that this script can run under other circumstances (perhaps in a debug mode, outside of this package.).